### PR TITLE
add-binlog post

### PR DESCRIPTION
**Pull request by buggithubs**

### CODE REVIEW

**Comment by sunhao2017** [comment]
LGTM

**Comment by andelf** [comment]
For newly added pictures, use `png` format.

File: op-guide/tidb-binlog.md [hunk 1]
```diff
@@ -0,0 +1,323 @@
+---
+title: TiDB-Binlog 部署方案
+category: advanced
+---
+
+
+
+# TiDB-Binlog 部署方案
```
Comment (andelf) [inline op-guide/tidb-binlog.md]: 1 empty line is ok

File: op-guide/tidb-binlog.md [hunk 2]
```diff
@@ -0,0 +1,323 @@
...
+
+## 目录
```
Comment (andelf) [inline op-guide/tidb-binlog.md]: conflicts with PDF output. better remove?

File: op-guide/tidb-binlog.md [hunk 3]
```diff
@@ -0,0 +1,323 @@
...
+  - [整体监控框架概述](https://github.com/pingcap/docs-cn/blob/master/op-guide/monitor-overview.md)
+  - [组件状态 API & 监控](#tidb-binlog-监控)
+
+## TiDB-Binlog 简介
+
+TiDB-Binlog 用于收集 TiDB 的 Binlog，并提供实时备份和同步功能的商业工具。
+
+TiDB-Binlog 支持以下功能场景:
+
+* *数据同步*:       同步 TiDB 集群数据到其他数据库
```
Comment (andelf) [inline op-guide/tidb-binlog.md]: better use table or list

File: op-guide/tidb-binlog.md [hunk 4]
```diff
@@ -0,0 +1,323 @@
...
+
+首先介绍 TiDB-Binlog 的整体架构。
+
+![TiDB-Binlog 架构](../media/architecture.jpeg)
+
+TiDB-Binlog 集群主要分为两个组件：
+
+### Pump 
+
+Pump 是一个守护进程，在每个 TiDB 的主机上后台运行。他的主要功能是实时记录 TiDB 产生的 Binlog 并顺序写入磁盘文件
```
Comment (andelf) [inline op-guide/tidb-binlog.md]: use 它 or 她 for a lovely product. 😄

File: op-guide/tidb-binlog.md [hunk 5]
```diff
@@ -0,0 +1,323 @@
...
+
+### Drainer
+
+Drainer 从各个 Pump 节点收集 Binlog，并按照在 TiDB 中事务的提交顺序转化为指定数据库兼容的 SQL 语句，最后同步到目的数据库或者写到顺序文件
```
Comment (andelf) [inline op-guide/tidb-binlog.md]: ambiguity here? What is 顺序文件's format, binary or plain sql?

File: op-guide/tidb-binlog.md [hunk 6]
```diff
@@ -0,0 +1,323 @@
...
+
+## TiDB-Binlog 安装
+
+### 下载官方 Binary
+
+- ( CentOS 7+ )
```
Comment (andelf) [inline op-guide/tidb-binlog.md]: single list item is unnecessary. A paragraph is ok

File: op-guide/tidb-binlog.md [hunk 7]
```diff
@@ -0,0 +1,323 @@
...
+sha256sum -c tidb-binlog-latest-linux-amd64.sha256
+
+# 解开压缩包
+tar -xzf tidb-binlog-latest-linux-amd64.tar.gz
+cd tidb-binlog-latest-linux-amd64
+```
+
+### TiDB-Binlog 部署
+
+- 推荐使用 ansible 部署 PUMP
```
Comment (andelf) [inline op-guide/tidb-binlog.md]: use Pump(or pump)

Comment (andelf) [inline op-guide/tidb-binlog.md]: normally, title-case for a product name, lower-case for the command.

File: op-guide/tidb-binlog.md [hunk 8]
```diff
@@ -0,0 +1,323 @@
...
+
+*   需要为一个 TiDB 集群中的每台 TiDB server 部署一个 pump，目前 TiDB server 只支持以 unix socket 方式的输出 binlog。
+
+    我们设置 TiDB 启动参数 binlog-socket 为对应的 pump 的参数 socket 所指定的 unix socket 文件路径，最终部署结构如下图所示：
+
+    ![TiDB pump 模块部署结构](../media/tidb_pump_deployment.jpeg)
+
+*   drainer 不支持对 ignore schemas（在过滤列表中的 schemas） 的 table 进行 rename DDL 操作
+
+*   在已有的 TiDB 集群中启动 drainer，一般需要全量备份 并且获取 savepoint，然后导入全量备份，最后启动 drainer 从 savepoint 开始同步；
```
Comment (andelf) [inline op-guide/tidb-binlog.md]: a space typo?

File: op-guide/tidb-binlog.md [hunk 9]
```diff
@@ -0,0 +1,323 @@
...
+    disable-dispatch = true
+
+    [syncer.to]
+    dir = "/path/pb-dir"
+    ```
+
+
+### 示例及参数解释
+
+1.  Pump
```
Comment (andelf) [inline op-guide/tidb-binlog.md]: too many lines in a single ordered list item.

Is level 4 title suitable here?

``####``

File: op-guide/tidb-binlog.md [hunk 10]
```diff
@@ -0,0 +1,323 @@
...
+    user = "root"
+    password = ""
+    port = 3306
+
+    # db-type 设置为 pb 时,存放 binlog 文件的目录
+    # [syncer.to]
+    # dir = "data.drainer"
+    ```
+
+-----
```
Comment (andelf) [inline op-guide/tidb-binlog.md]: useless hr?

**Pull request merged**